### PR TITLE
fix(harvest): Remove unsubscribingAll for realtime

### PR DESCRIPTION
**Pull request by CPatchane**
We will have to unsubscribe at term to handle correctly not used Realtime instances, but for now this unsubscribeAll just close the Job real-time instance and makes it unusable

### CODE REVIEW

File: packages/cozy-harvest-lib/src/models/KonnectorJob.js [hunk 1]
```diff
@@ -150,21 +152,32 @@ export class KonnectorJob {
...
+
+    this.unsubscribeAllRealime = () => {
+      jobWatcher.unsubscribeAll()
+      accountWatcher.unsubscribeAll()
+    }
+  }
+
+  onUnmount() {
+    if (typeof this.unsubscribeAllRealime === 'function') {
+      this.unsubscribeAllRealime()
```
Comment (edas) [inline packages/cozy-harvest-lib/src/models/KonnectorJob.js]: `this.unsubscribeAllRealtime()`

File: packages/cozy-harvest-lib/src/models/KonnectorJob.js [hunk 2]
```diff
@@ -150,21 +152,32 @@ export class KonnectorJob {
...
     })
+
+    this.unsubscribeAllRealime = () => {
+      jobWatcher.unsubscribeAll()
+      accountWatcher.unsubscribeAll()
+    }
+  }
+
+  onUnmount() {
+    if (typeof this.unsubscribeAllRealime === 'function') {
```
Comment (edas) [inline packages/cozy-harvest-lib/src/models/KonnectorJob.js]: `if (typeof this.unsubscribeAllRealtime === 'function') {`

File: packages/cozy-harvest-lib/src/models/KonnectorJob.js [hunk 3]
```diff
@@ -150,21 +152,32 @@ export class KonnectorJob {
...
-    watchKonnectorAccount(this.account, {
+    const accountWatcher = watchKonnectorAccount(this.account, {
       onTwoFACodeAsked: this.handleTwoFA,
-      onLoginSuccess: konnectorJob.handleSuccess,
-      onLoginSuccessHandled: konnectorJob.disableSuccessTimer
+      onLoginSuccess: jobWatcher.handleSuccess,
+      onLoginSuccessHandled: jobWatcher.disableSuccessTimer
     })
+
+    this.unsubscribeAllRealime = () => {
```
Comment (edas) [inline packages/cozy-harvest-lib/src/models/KonnectorJob.js]: ` this.unsubscribeAllRealtime = () => {`

File: packages/cozy-harvest-lib/src/models/KonnectorJob.js [hunk 4]
```diff
@@ -47,6 +48,7 @@ export class KonnectorJob {
     this.handleTwoFA = this.handleTwoFA.bind(this)
     this.launch = this.launch.bind(this)
     this.sendTwoFACode = this.sendTwoFACode.bind(this)
+    this.onUnmount = this.onUnmount.bind(this)
```
Comment (gregorylegarec) [inline packages/cozy-harvest-lib/src/models/KonnectorJob.js]: KonnectorJob should not have any knowledge about React components lifecycle. So this method should not be named after a React concept like unmounting.

I suggest instead `unwatch()`.

**Pull request merged**